frontend/device/generic/device.lua

```diff
@@ -46,6 +46,9 @@ local Device = {
     isAlwaysPortrait = no,
     -- needs full screen refresh when resumed from screensaver?
     needsScreenRefreshAfterResume = yes,
+    -- framebuffer reports 8bit per pixel, but is actually 24bit per pixel
```

Comment:
I'd add a link to your original issue in the comment, in case that confuses someone down the road ;).

Reply:
Possibly even saying right then and there that it's a Triton/color eInk screen